apache/incubator-zeppelin
# JDBC generic interpreter

## What is this PR for?

JDBC generic interpreter based on PostgresInterpreter (@tzolov)

## What type of PR is it?

Feature

## Is there a relevant Jira issue?
## How should this be tested?
In the file zeppelin-daemon.sh add:
ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/jdbc/jars/jdbc jar"

add interpreter and properties. See screenshots.

## Screenshots (if appropriate)
zeppelin-daemon.sh add jars
![image](https://cloud.githubusercontent.com/assets/10614247/12169690/a576f3be-b53a-11e5-9c63-d269038a7b5e.png)

Interpreter properties (@jongyoul thanks)
![image](https://cloud.githubusercontent.com/assets/10614247/12169645/4f395b9a-b53a-11e5-8a2c-326ea1758832.png)


for use in paragraph:

![image](https://cloud.githubusercontent.com/assets/10614247/12169963/07bc5c9c-b53d-11e5-80e1-72da6fc8c345.png)


## Questions:

It's finished? no,we will review it and discuss it
Does the licenses files need update? no
Is there breaking changes for older versions? no
Does this needs documentation? documentation will follow
